tests/nlu/data_augmentation/test_augmenter.py

```diff
@@ -0,0 +1,618 @@
+import operator
+import os
+from typing import Callable, List, Set, Text
+
+import pytest
+from rasa.nlu.constants import TOKENS_NAMES
+from rasa.nlu.tokenizers.tokenizer import Tokenizer
+from rasa.constants import DEFAULT_RANDOM_SEED
+from rasa.shared.nlu.training_data.message import Message
+import rasa.nlu.config
+import rasa.shared.nlu.training_data.loading
+import rasa.shared.utils.components
+import rasa.shared.utils.io
+from rasa.shared.nlu.constants import INTENT, TEXT, VOCABULARY, INTENT_REPORT_FILE_NAME
+
+
+def test_augmenter_create_tokenizer():
```

Comment:
this test should go into a file `rasa.shared.utils.test_components` as it's solely testing `get_tokenizer_from_nlu_config`

Reply:
(thanks for writing this test! 💯 )

Reply:
So, do you think we can remove the test altogether though? Its not testing any functionality that has been introduced by data augmentation. As you say it boils down to loading a tokenizer from a (valid) config, which is already handled by dedicated tests.

Reply:
Now that we have it, I wouldn't remove it. Which dedicated tests do you mean? From what I saw (just gave it a short glimpse), it might be tested implicitly but not by a dedicated test.